platforms/documentation/docs/src/snippets/daemon-jvm/customized-task/groovy/settings.gradle

```diff
@@ -1 +1,6 @@
+plugins {
+    // Apply the foojay-resolver plugin to allow automatic download of JDKs
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
```

Comment:
🤔  We've migrated our build and all samples to `0.9.0` recently
- https://github.com/gradle/gradle/pull/31970